Tribler/Core/SessionConfig.py

```diff
@@ -608,6 +608,48 @@ def set_enable_channel_search(self, mode):
         """
         self.sessconfig.set(u'allchannel_community', u'enabled', mode)
 
+    def get_enable_metadata_community(self):
```

Comment:
@NielsZeilemaker I will probably just use the Channel community for metadata propagation. This PR is to fix the metadata download via TFTP only so I haven't modified the metadata community part yet.